.github/ISSUE_TEMPLATE/rule_deprecation.yaml

```diff
@@ -0,0 +1,21 @@
+name: Rule Deprecation
+description: Recommendation to deprecate a rule
+title: "[Deprecation] Name of the rule"
```

Comment:
We have an undocumented workflow for this since saved objects are STILL not removed instances if missing from a package. Typically the following is done.

- Rule tuning where `Deprecated - ` is prepended to the rule name
- Rule tuning is merged and goes out in the next release
- Rule deprecation is opened to deprecate the rule officially
- Rule deprecation is merged

@approksiu

Reply:
ill add this in the deprecation guidelines.